### DISCUSSION TITLE
Dogecoin Node as a consumer friendly product

### DISCUSSION THREAD

**Discussion by DogeWhisper** [veryengineer/collab#12] [Ideas]
The thought behind this is to make a consumer themed product that is easy to use, would have a demand and reach the masses. At the same time it would help the Doge network.

A Dogecoin themed alarm clock that projects time on wall, reads out loud your morning tweets, has a charging dock for phone & most importantly is a Dogecoin Node

A price target of $69 would be great too 😂

Updated thought: Maybe skip the speaker and project latest tweets of those you follow on wall with clock (time)

Who can help make it happen? Any engineers / devs / entrepreneurs ?

![FoIn-YCWAAM3W-6](https://user-images.githubusercontent.com/123888437/216779546-6707f27e-f4a4-4c00-ae2c-e6ea5e429239.jpg)

**Comment by Doge10420** [comment]
I think 3D printers can print this easy… first make 2-3 to make sure people would like to buy

**Reply by DogeWhisper** [reply]
3d printers can probably be used for the concept. I think the hardest part would be the software and the hardware inside. What would be needed to design? What is feasibly possible so cost remains low? What would it all cost to source and create? This is what we have to figure out

**Comment by Doge369420** [comment]
Great idea! Would just need to find out a minimum requirements for SBC and HDD with software that can automate node and whatever other things you want like a display and speaker. 
Once a prototype is done then can do a custom pcb or whatever, possibly even approach a third party producer for quotes to complete whole units at a scale far cheaper overseas.

**Reply by Doge369420** [reply]
Maybe chat bot can code it for us? 🤣

**Reply by DogeWhisper** [reply]
Don't laugh.. I tried lol! I think we need to call in the experts tho 

To build an alarm clock with the features specified, the following electrical components would be required:

    Power adapter: To provide power to the device
    Microcontroller: To control the projector and charging dock, as well as play audio through the speaker
    Projector: To project the time on the wall
    Charging dock: To charge the phone
    Speaker: To read out loud tweets
    Display: To display the time and other information
    Network interface (such as Wi-Fi or Ethernet): To connect to the internet and fetch tweets
    Voltage regulators: To regulate the power supplied to the microcontroller and other components to ensure stable operation
    Capacitors: To smooth out power fluctuations
    Resistors: To regulate the current flowing in the circuits
    Hard Drive: To store Dogecoin blockchain 
    Connectors and wiring: To connect the components and form the electrical connections

It may also require additional components such as switches, transistors, and diodes, depending on the design and implementation of the device.

**Reply by Doge369420** [reply]
Nice work, halfway there already.. Maybe if they see us trying they will help us? ;D

Lots of prototyping done on Raspberry Pi, I see in other thread they doing node for the RPi. 
That is part of it done. 

Not sure how many watts it can handle but most of can be done with existing parts made for RPi. 
Maybe some bits here and there can borrow from other projects online for now, surely there is code for radio, it has wifi and Ethernet built in, GPIO pins. 

Charging ports - USB and GPIO pins
Connectivity - Wifi, Ethernet and Bluetooth 
Display - various displays including touch screen 
Projector - USB or GPIO
Speaker - USB or GPIO
Most of the hard work would be coding it though 😅

**Comment by daank-c** [comment]
YES. Let's do this one!

I have intermediate experience with consumer electronics, drawing/sourcing the proper schematics, and have some idea of the components req. to make this all work. But I'm not very good with software (although I do dabble) so the mini-node thing would be a bit out of my realm.

**Comment by DogestMaximus** [comment]
We started working on this and were just wondering what other ideas anyone may have.  There is an existing thread on the bird [HERE](https://twitter.com/DogestMaximus/status/1627725797925453836?s=20) but for the sake of simplicity if we could keep it on this Github thread that would be easier to read for everyone to contribute who is interested.  Some sample ideas were: NAS/RAID controller, Gaming Console/Media server, and from the spec sheet: 
![image](https://user-images.githubusercontent.com/119521071/220176358-3f370512-24d1-4728-a45b-5015df2bb03c.png)

If you have ideas you want to add this is the place.

Here's what we are working with: http://download.t-firefly.com/%E4%BA%A7%E5%93%81%E8%A7%84%E6%A0%BC%E6%96%87%E6%A1%A3/ROC-RK3328-CC_Product%20Specifications%20V1.0.pdf

**Reply by DogestMaximus** [reply]
@Doge_France had an idea for a loose change > doge converter: https://twitter.com/Doge_France/status/1641485749101920256?s=20

We have a way to run just about any distribution including chromium os which may be a good POS (point of sale) jump off.  Any ideas?

**Reply by DogeWhisper** [reply]
Coinstar added the ability to covert change to a Doge but they are more of a vending machine type service than a POS
![565AC5F7-394F-4EB4-A07D-077B6281101D](https://user-images.githubusercontent.com/123888437/229112366-8de54d5b-b873-4f5e-87b8-7ba4eb438f21.jpeg)

**Comment by tosufever** [comment]
this is pretty cool for $20 at walmart.  

 
![image](https://user-images.githubusercontent.com/108495285/224465472-3ffc7a33-4466-4b11-8d49-470185c16333.png)

**Reply by DogestMaximus** [reply]
I've used the HDMI version and they are nice- there are some driver and configuration options to contend with but they do work well.

**Comment by K1ngsatoshee** [comment]
LFG!!!
